tests/alerts/geomodel/test_locality.py

```diff
@@ -0,0 +1,270 @@
+from datetime import datetime, timedelta
+import pytz
+from typing import Optional
+import unittest
+
+from mozdef_util.query_models import SearchQuery
+
+import alerts.geomodel.config as config
+import alerts.geomodel.locality as locality
+
+from tests.unit_test_suite import UnitTestSuite
+
+
+def query_interface(results: locality.Entry) -> locality.QueryInterface:
+    '''Produce a `QueryInterface` that just returns the provided results.
+    '''
+
+    def closure(q: SearchQuery, esi: str) -> Optional[locality.Entry]:
+        return results
+
+    return closure
+
+
+def _now() -> datetime:
+    return datetime.utcnow().replace(tzinfo=pytz.UTC)
```

Comment:
We should use `toUTC` https://github.com/mozilla/MozDef/blob/master/mozdef_util/mozdef_util/utilities/toUTC.py#L10 instead